### ISSUE TITLE
Oregon Washington boundary

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [pfmc-assessments/nwfscSurvey#85]
@okenk Recently asked me about separating survey data by state. I see in [getStrata.R](https://github.com/pfmc-assessments/nwfscSurvey/blob/e291e13beac13e28332e0706708bd53da644158f/R/GetStrata.fn.R#L50-L52) that we define the boundary between oregon and washington at 46, but a brief rabbit trail before finding that led me to the [Washington State Legislature](https://app.leg.wa.gov/rcw/default.aspx?cite=43.58.060) which defines it as 46 and 15 minutes (so 46.25). I couldn't find another way to distinguish by state in nwfscSurvey, but using 46.25 would seem more accurate.

**Comment by iantaylor-NOAA** [comment]
I have used 46.25 in other places (for sure in maps). However, the SA3 data file used by {nwfscSurvey} uses 0.5 degree breaks (see https://github.com/pfmc-assessments/nwfscSurvey/issues/38), so I presume that 46.0 seemed like a better way to round than 46.5. For purposes of composition expansions, I'm guessing that it will not make any visible difference, so not worth requesting a revised data file as we did in issue #38 to add 40°10' to the data.
